### ISSUE TITLE
Remove custom link color

### ISSUE THREAD

**Issue by rahulkgupta** [openoakland/openoakland.org#66]
## Description
The link color is the custom OO green. This causes confusion about whether elements are clickable or just bolded a certain color. It also [fails webAIM standards](https://webaim.org/resources/linkcontrastchecker/?fcolor=21B890&bcolor=FFFFFF&lcolor=198D6E).

## Potential solutions
- Original in issue: Moving the link color back to the default color will reduce confusion.
- For other options, see the Style Guide explorations: [OpenOakland Style Guide's typography section](https://docs.google.com/document/d/1SvhwrQcu4mNMS05zE4AIXA8zsc7qpUeWVA9GvgiZaHo/edit#bookmark=id.liliqlz4j4xq) for link styling explorations.

**Comment by theecrit** [comment]
Added the style guide recommendations to this issue for reference.
